src/primitives/decode_and_deserialize.m

```diff
@@ -0,0 +1,32 @@
+function deserialized_data = decode_and_deserialize(data, decompress)
```

Comment:
Why do you need this function?

If you pass **support_qiskit** to False, results are based on standard JSON